src/frontend/src/features/layouts/components/mailbox-panel/components/mailbox-labels/components/label-form-modal/index.tsx

```diff
@@ -36,6 +38,8 @@ export const LabelModal = ({ isOpen, onClose, label }: LabelModalProps) => {
       name: label?.display_name ?? '',
       color: label?.color ?? '#E3E3FD',
       parent_label: label?.name.split('/').slice(0, -1).join('/') ?? undefined,
+      description: label?.description ?? '',
+      is_enabled: label?.is_enabled ?? true,
```

Comment:
Not quit sure we want that by default. Furthermore this checkbox should be shown only if the auto labeling feature is enabled.